budaminof/vue-exercise
# F / Display Blog Posts

### What does this PR do?
1. This PR makes it possible to call the API to get the list of posts, filters them by user id and display both lists.

2. Dependencies: @vue/test-utils, axios-mock-adapter, karma-chrome-launcher

**Note** - there is a `setTimeout` on the API request to mock delay in the server to the show loading state.

### How to test this?
`npm i` to update dependencies.
`npm run dev` to test in the browser.
`npm run test` to see the test running.

![giphy](https://media.giphy.com/media/l0IycQmt79g9XzOWQ/giphy.gif)
